alerter/alerter.cs

```diff
@@ -1,31 +1,53 @@
-﻿using System;
+using System;
+using System.Diagnostics;
 
-namespace AlerterSpace {
-    class Alerter {
+namespace AlerterSpace
+{
+    class Alerter
+    {
         static int alertFailureCount = 0;
-        static int networkAlertStub(float celcius) {
-            Console.WriteLine("ALERT: Temperature is {0} celcius", celcius);
-            // Return 200 for ok
-            // Return 500 for not-ok
-            // stub always succeeds and returns 200
+        static int networkAlertStub(float celcius)

```

Comment:
Stub is part of production code. This needs to be separated out. Keep production code and test code separate.

Use abstractions like interfaces or method delegates to abstract test code/unavailable dependencies in the production code.